Lib/test/test_shutil.py

```diff
@@ -1101,6 +1101,36 @@ def test_copytree_subdirectory(self):
         rv = shutil.copytree(src_dir, dst_dir)
         self.assertEqual(['pol'], os.listdir(rv))
 
+    def test_copytree_to_itself_gives_sensible_error_message(self):
+        base_dir = self.mkdtemp()
```

Comment:
You need to add some cleanups to delete the files at the end. See some of the other tests for reference.

Reply:
fixed.